packages/reporter/README.md

```diff
@@ -0,0 +1,24 @@
+# @secbox/reporter
+
+Provide an abstraction for generating test results as part of the particular test frameworks.
+
+## Setup
+
+```bash
+npm i -s @secbox/reporter
+```
+
+## Usage
+
+```ts
+import { Reporter, StdReporter } from '@secboox/reporter';
+
+const reporter: Reporter = new StdReporter();
+await reporter.report(scan);
+```
```

Comment:
`````suggestion
import { Reporter, StdReporter } from '@secboox/reporter';

const reporter: Reporter = new StdReporter();
```
You just need to call the `report` method to send findings to stdout:

```ts
await reporter.report(scan);
```
`````

I guess it might be helpful to provide a screenshot of the final report in stdout as well.